src/main/java/org/prebid/server/handler/SetuidHandler.java

```diff
@@ -104,12 +105,22 @@ public SetuidHandler(long defaultTimeout,
         this.metrics = Objects.requireNonNull(metrics);
         this.timeoutFactory = Objects.requireNonNull(timeoutFactory);
 
-        cookieNameToSyncType = bidderCatalog.names().stream()
+        cookieNameToSyncType = collectMap(bidderCatalog);
+    }
+
+    private static Map<String, UsersyncMethodType> collectMap(BidderCatalog bidderCatalog) {
+
+        final List<Usersyncer> usersyncers = bidderCatalog.names().stream()
                 .filter(bidderCatalog::isActive)
                 .map(bidderCatalog::usersyncerByName)
                 .filter(Optional::isPresent)
                 .map(Optional::get)
-                .distinct() // built-in aliases looks like bidders with the same usersyncers
+                .distinct()
```

Comment:
I'd leave the comment intact. Might help someone in the future.

Reply:
Got it.